### ISSUE TITLE
chineseanime.org: detection

### ISSUE THREAD

**Issue by CarolloSenpai** [uBlockOrigin/uAssets#18227]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.chineseanime.org/`

### Category

detection

### Description

![Zrzut ekranu 2023-05-25 093632](https://github.com/uBlockOrigin/uAssets/assets/134587331/aa0126f2-4226-4bbc-8721-e5d02281d07d)

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.49.2
Chromium: 113
filterset (summary): 
 network: 96501
 cosmetic: 46257
 scriptlet: 17656
 html: 0
listset (total-discarded, last-updated): 
 default: 
  user-filters: 0-0, never
  POL-0: 7943-11, 46m
  POL-2: 1275-13, 1d.17h.9m
  easylist: 65952-72, 5d.19h.34m
  easyprivacy: 31838-46, 3d.9h.19m
  plowe-0: 3735-946, 22h.53m
  ublock-abuse: 70-0, 22h.52m
  ublock-badware: 7098-127, 4d.10h.18m
  ublock-filters: 33017-134, 3d.13h.44m
  ublock-privacy: 392-74, 22h.51m
  ublock-quick-fixes: 336-38, 45m
  ublock-unbreak: 2020-41, 4d.10h.19m
  urlhaus-1: 8396-1, 47m
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
 allReadyAfter: 190 ms (selfie)
 maxAssetCacheWait: 109 ms
popupPanel: 
 blocked: 8
 network: 
  googlesyndication.com: 4
  googletagmanager.com: 1
  histats.com: 1
  onesignal.com: 2
 cosmetic: 
  ##ins.adsbygoogle
  ##[id^=\"aswift_\"]
  #@#[data-ad-module]
  #@#[data-ad-width]
  #@#[data-adblockkey]
  #@#[data-advadstrackid]
  #@#[data-ad-manager-id]
```
</details>

**Comment by CarolloSenpai** [comment]
There is no fix for it ?

**Comment by mapx-** [comment]
`closed this as completed in 9c062f9 5 hours ago`

This means it was closed after been fixed.
Click `9c062f9` above and you'l see the commit

Now you have to update your filter lists in uBO's dashboard.
